nc-news-project/src/components/singleArticle.js

```diff
@@ -55,14 +76,39 @@ const SingleArticle = () => {
                     <p id="article-body">{article.body}</p>
 
                     <p id="article-votes"> 
-                    <button id="vote-button" onClick={handleUpVote} disabled={voteChange>0}>{'+'}</button>
+                    <button id="vote-button" onClick={handleUpVote} disabled={voteChange>0 || userObj.username === article.author}>{'+'}</button>
                     Votes: {votes + voteChange}
-                    <button id="vote-button" onClick={handleDownVote} disabled={voteChange < 0}>{'-'}</button></p>
+                    <button id="vote-button" onClick={handleDownVote} disabled={voteChange < 0 || userObj.username === article.author}>{'-'}</button></p>
 
                     <p id="article-comments">Comments: {article.comment_count}</p>
                 </li>
         </ul>
+
+        <div className="article-break">
         <h3 id="comment-title">Comments</h3>
+        <button id="new-commentButton" disabled={userObj.username === article.author} onClick={handleNewComment}>New Comment</button>
+        </div>
+        {newCommentCheck && (
+            <form className="new-comment">
+                <input 
+                id="new-commentInput" 
+                autoComplete="off" 
+                onChange={(e) => {
+                    setNewComment(e.target.value)
+                }} 
+                placeholder="Type..."
+                value={newComment}>
+                </input>
+                <input 
+                id="new-commentSubmit" 
+                onClick={handleSubmit}
```

Comment:
We should instead do this as an onSubmit on your form element because you cannot currently post a comment as a keyboard only user. Hitting enter causes a refresh because the form is submitting and you are not preventing the default behaviour of it.